integration/simulation/utils.go

```diff
@@ -53,6 +61,28 @@ func createInMemObscuroNode(id int64, genesis bool, avgGossipPeriod uint64, avgB
 	return &node
 }
 
+func createStandaloneObscuroNode(id int64, genesis bool, avgGossipPeriod uint64, stats *Stats, peerAddrs []string) *host.Node {
```

Comment:
these are not really standalone. (started in a separate process)
It's more like the communication is more realistic

Reply:
Will fix, thanks